DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/SimpleInjectorConfiguration.cs

```diff
@@ -28,7 +20,8 @@ public static Container RegisterPackages(Container container)
         {
             container.RegisterPackages(new[]
             {
-                typeof(DALPackage).Assembly,
+                typeof(WebApiActivityPackage).Assembly,
+                typeof(DALPackage).Assembly, 
```

Comment:
DALPackage, is it informative? About what dalpackage you are meant? activity? notes?